resources/lang/en/patrols/forest/border/any.json

```
@@ -244,7 +277,10 @@
                         "cats_from": ["clan", "patrol"],
```

```suggestion
                        "cats_from": ["clan"],
```
this is a little quirk i've noticed a few times! patrol should just be removed in this instance since they're included as part of "clan"